ethers-core/src/types/chain.rs

```diff
@@ -92,9 +92,9 @@ pub enum Chain {
 
     Metis = 1088,
 
-    #[strum(to_string = "xdai", serialize = "gnosis", serialize = "gnosis-chain")]
-    #[serde(alias = "xdai", alias = "gnosis", alias = "gnosis_chain")]
-    XDai = 100,
+    #[strum(to_string = "gnosis", serialize = "gnosis-chain")]
+    #[serde(alias = "gnosis", alias = "gnosis_chain")]
+    Gnosis = 100,
```

Comment:
Can you keep `"xdai"` as an alias on both strum and serde to avoid a breaking change?

Reply:
Oh sorry, absolutely. Addressed in ab0071ba4f63d1db346fde773be0b14e25c32028